stubs/inertia/resources/js/Pages/API/ApiTokenManager.vue

```diff
@@ -227,9 +227,10 @@
             createApiToken() {
                 this.createApiTokenForm.post(route('api-tokens.store'), {
                     preserveScroll: true,
-                }).then(response => {
-                    if (! this.createApiTokenForm.hasErrors()) {
-                        this.displayingToken = true
+                    onSuccess: () => {
+                        if (! this.createApiTokenForm.hasErrors()) {
+                            this.displayingToken = true
+                        }
```

Comment:
This is still called `onSuccess` even if there are validation errors? I guess because it's a redirect on the server side?

Reply:
Yeah, it does.. `onSuccess` in Inertia doesn't have much to do with the HTTP request itself succeeding, but rather that a valid (usable) Inertia response came back. https://inertiajs.com/manual-visits#promise-deprecation

Reply:
Yes, exactly. Success = successful Inertia visit. It's like a `then()` before.